src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -42,7 +42,7 @@ If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node
 
 ### Archive node {#archive-node}
 
-- Stores everything kept in the full node and builds an archive of historical states. Needed if you want to query something like an account balance at block #4,000,000.
+- Stores everything kept in the full node and builds an archive of historical states. Needed if you want to query something like an account balance at block #4,000,000 or simply and realiablely [test your own transactions set without mining them using OpenEthereum](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany).
```

Comment:
- Would add comms after `#4,000,000`
- "realiablely" -> "reliably"

Reply:
I approved you for doing the commit.